### ISSUE TITLE
Not full Working for Visual Studio 2015

### ISSUE THREAD

**Issue by rjpeck2** [justcla/HotSettings#10]
Most of the commands are not available for Visual Studio 2015.

**Comment by justcla** [comment]
Which ones do you use the most?
I'll try to get them done first.

**Comment by rjpeck2** [comment]
I use both of them. Don’t rush on my account.

Thanks

**Comment by justcla** [comment]
No, I mean, which settings in Hot Settings do you want me to expose in VS2015?

**Comment by gundermanc** [comment]
It probably is failing to resolve the implementation for [IVsTextManager6](https://github.com/justcla/HotSettings/blob/master/HotSettings/VSInternal/IVsTextManager6.cs). I believe it was introduced in Visual Studio 2017 with the Structure Guides feature.

To get this to work on 2015, you'll probably need to use an earlier revision of this API for everything except the structure guides setting.

**Comment by justcla** [comment]
Yeah, figured as much. I saw this coming when we switched to the new API.
If everything else will work with an older IVsTextManager, then I guess we should drop back to that.

IVsTextManager4 ?

**Comment by gundermanc** [comment]
Tbh I haven't debugged it at all, I'm just guessing wildly :)

I'd give that a try

**Comment by ZAIDI123** [comment]
// Creates a folder that anyone on the Internet can read from and write to. (Domain
// administrators can prohibit this setting for G Suite users.)
var folder = DriveApp.createFolder('Shared Folder');
folder.setSharing(DriveApp.Access.ANYONE, DriveApp.Permission.EDIT);
